index.html

```diff
@@ -50,13 +50,17 @@
   <!-- End Matomo Code -->
 </head>
 <body>
+	<div class="language-menu">
+		<button class="language-menu--btn">English ▼</button>
+		<a class="language-menu--item" href="/es/index.html">Spanish</a>
```

Comment:
Remember, we don't need to include `index.html` in links. The server knows how to read the files. `/es` automatically serves the index file in a folder.